spec/cc/config/validation/json_spec.rb

```diff
@@ -219,6 +220,17 @@
     expect(validator.warnings).to include("unrecognized key 'exclude_pattttttterns'")
   end
 
+  describe "version validation" do
+    it "warns about missing version" do
+      validator = validate_json(<<-EOJSON)
```

Comment:
TIL that the closing parenthesis can go here! I've always placed it after the closing heredoc.